yt/geometry/grid_geometry_handler.py

```diff
@@ -450,6 +450,25 @@ def _chunk_io(
                 with self.io.preload(dc, preload_fields, 4.0 * size):
                     yield dc
 
+    def _icoords_to_fcoords(self, icoords, ires, axes=None):
+        """
+        Accepts icoords and ires and returns appropriate fcoords and fwidth.
+        Mostly useful for cases where we have irregularly spaced or structured
+        grids.
+        """
+        dds = self.ds.domain_width[axes,] / (
```

Comment:
I think this is what flake8 complains about
```suggestion
        dds = self.ds.domain_width[axes] / (
```

Reply:
Hm, I think at the time I wrote it, I *thought* I was changing things by using the comma, but I suppose not.  Once the tests are done running I'll do this, so that we can at least see if the current run will pass.

Reply:
I convinced myself they are perfectly equivalent:
```python
>>> import numpy as np
>>> a = np.arange(10)
>>> a[0]
0
>>> a[0,]
0
>>> a[0, ...]
array(0)
>>> a[0, :]
...
IndexError: too many indices for array: array is 1-dimensional, but 2 were indexed
```